packages/testsuite/cypress.config.ts

```diff
@@ -30,50 +34,156 @@ export default defineConfig({
+                            resolve(`http://localhost:${managementPortWithOffset}`);
                           }
                         })
-                        /* eslint @typescript-eslint/no-empty-function: off */
-                        .catch(() => {});
-                    }, 500);
-                  });
+                        .catch((error) => {
+                          console.log(error);
+                        });
+                    });
+                } else {
+                  startedContainersManagementPorts.set(name as string, wildflyContainer.getMappedPort(9990));
+                  const managementApi = `http://localhost:${wildflyContainer.getMappedPort(9990)}/management`;
+                  return axios
+                    .post(managementApi, {
+                      operation: "list-add",
+                      address: ["core-service", "management", "management-interface", "http-interface"],
+                      name: "allowed-origins",
+                      value: `http://localhost:${config.env.HAL_CONTAINER_PORT as string}`,
+                    })
+                    .then(() => {
+                      return axios.post(managementApi, {
+                        operation: "reload",
+                      });
+                    })
+                    .then(() => {
+                      const startTime = new Date().getTime();
+                      const interval = setInterval(() => {
+                        if (new Date().getTime() - startTime > 10000) {
+                          clearInterval(interval);
+                          reject();
+                        }
+                        axios
+                          .post(managementApi, {
+                            operation: "read-attribute",
+                            name: "server-state",
+                          })
+                          .then((response) => {
+                            if ((response as { data: { result: string } }).data.result == "running") {
+                              clearInterval(interval);
+                              const wildflyServer = `http://localhost:${wildflyContainer.getMappedPort(9990)}`;
+                              resolve(wildflyServer);
+                            }
+                          })
+                          .catch((error) => {
+                            console.log(error);
+                          });
+                      }, 500);
+                    });
+                }
               })
-              .catch((err) => reject(err));
+              .catch((err) => {
+                console.log(err);
+                reject(err);
+              });
           });
         },
+        "start:keycloak:container": ({ name }) => {
+          return findAPortNotInUse(8888, 8988)
+            .then((freePort: number) => {
+              startedContainersManagementPorts.set(name as string, freePort);
```

Comment:
I believe this is not needed for Keycloak containers

Reply:
You are right. removed also with unnecessary waiting block for premise which caused requirement for this map.